pkgs/applications/virtualization/docker/default.nix

```
@@ -45,6 +45,10 @@ rec {
         hash = runcHash;
       };
 
+      preBuild = ''
+        substituteInPlace Makefile --replace-fail "/bin/bash" "${bash}/bin/bash"
```

```suggestion
        substituteInPlace Makefile --replace-quiet "/bin/bash" "${bash}/bin/bash"
```
This breaks builds on v25 and v26

Could also use --replace-warn. Up to you